config/chime_science_run_gpu.yaml

```diff
@@ -1354,6 +1332,76 @@ rfi_sk_record:
 #    rfi_in: gpu_rfi_var_output_buffer_3
 #    updatable_config: "/updatable_config/rfi_var_record"
 
+#### Monitoring and debug stages ####
+
+buffer_status:
+  kotekan_stage: bufferStatus
+  time_delay: 30000000
+  print_status: false
+
+monitor:
+  kotekan_stage: monitorBuffer
+  bufs:
+    - network_buffer_0
+    - network_buffer_1
+    - network_buffer_2
+    - network_buffer_3
+  timeout: 5
+  fill_threshold: 0.98
+
+# This seems to find the lockup condition
+monitor_2:
+  kotekan_stage: monitorBuffer
+  bufs:
+    - beamform_pulsar_output_buffer_0
+    - beamform_pulsar_output_buffer_1
+    - beamform_pulsar_output_buffer_2
+    - beamform_pulsar_output_buffer_3
+    - lost_samples_buffer
+  timeout: 10
+  fill_threshold: 0.80
+
+
+############################
+#### Fixed reorder maps ####
+############################
+
+# FRB reorder map
+# TODO: can this be generated from teh input_reorder data below?
```

Comment:
```suggestion
# TODO: can this be generated from the input_reorder data below?
```

Reply:
I would guess it could be generated from the list below, but it's not actually clear what has gone into this list, and I don't see an obvious correspondence.